pino.js

```diff
@@ -22,6 +23,8 @@ var defaultOptions = {
   safe: true,
   name: undefined,
   serializers: {},
+  redact: [],
+  censor: '[Redacted]',
```

Comment:
How about `censorWith`?

Reply:
Or `redactWith`.

Reply:
given the context it seems to make the same amount of sense a `censor` does, 
I think it's self explanatory - what else would `censor: <String>` mean for a logger?

Reply:
To me, `censor` implies a boolean that would enable or disable censoring.